cmd/status-cli/main.go

```diff
@@ -0,0 +1,347 @@
+					}
+					defer func() { _ = aliceMessenger.Shutdown() }()
+
+					aliceID := types.EncodeHex(crypto.FromECDSAPub(aliceMessenger.IdentityPublicKey()))
+					fmt.Println("[Alice] messenger started, id:", aliceID)
+
+					time.Sleep(3 * time.Second)
+
+					// Start bob node and messenger
+
+					fmt.Println("[Bob] starting messenger")
+					bobPrivKey, err := crypto.GenerateKey()
+					if err != nil {
+						fmt.Println(err)
+						return err
+					}
+
+					bobConfig := &wakuv2.Config{}
+					bobConfig.EnableDiscV5 = true
+					bobConfig.DiscV5BootstrapNodes = []string{enrBootstrap}
+					bobConfig.DiscoveryLimit = 20
+					bobNode, err := wakuv2.New("", "", bobConfig, nil, nil, nil, nil, nil)
+					if err != nil {
+						fmt.Println(err)
+						return err
+					}
+
+					err = bobNode.Start()
+					if err != nil {
+						fmt.Println(err)
+						return err
+					}
+					defer func() { _ = bobNode.Stop() }()
+
+					time.Sleep(3 * time.Second)
+
+					appDb2, err := helpers.SetupTestMemorySQLDB(appdatabase.DbInitializer{})
+					if err != nil {
+						fmt.Println(err)
+						return err
+					}
+					walletDb2, err := helpers.SetupTestMemorySQLDB(walletdatabase.DbInitializer{})
+					if err != nil {
+						return err
+					}
+					madb2, err := multiaccounts.InitializeDB(dbsetup.InMemoryPath)
+					if err != nil {
+						return err
+					}
+					acc2 := generator.NewAccount(bobPrivKey, nil)
+					iai2 := acc2.ToIdentifiedAccountInfo("")
+					bobOptions := []protocol.Option{
+						protocol.WithCustomLogger(nil),
+						protocol.WithDatabase(appDb2),
+						protocol.WithWalletDatabase(walletDb2),
+						protocol.WithMultiAccounts(madb2),
+						protocol.WithAccount(iai2.ToMultiAccount()),
+						protocol.WithDatasync(),
+						protocol.WithToplevelDatabaseMigrations(),
+						protocol.WithBrowserDatabase(nil),
+					}
+					bobMessenger, err := protocol.NewMessenger(
+						"bob-node",
+						bobPrivKey,
+						gethbridge.NewNodeBridge(nil, nil, bobNode),
+						uuid.New().String(),
+						nil,
+						bobOptions...,
+					)
+					if err != nil {
+						fmt.Println(err)
+						return err
+					}
+
+					err = bobMessenger.Init()
+					if err != nil {
+						fmt.Println(err)
+						return err
+					}
+
+					_, err = bobMessenger.Start()
+					if err != nil {
+						fmt.Println(err)
+						return err
+					}
+					defer func() { _ = bobMessenger.Shutdown() }()
+
+					bobID := types.EncodeHex(crypto.FromECDSAPub(bobMessenger.IdentityPublicKey()))
+					fmt.Println("[Bob] messenger started, id:", bobID)
+
+					time.Sleep(3 * time.Second)
+
+					// Send contact request from Alice to Bob
+
+					fmt.Println("[Alice] send contact request to bob, contact id:", bobID)
+					request := &requests.SendContactRequest{
+						ID:      bobID,
+						Message: "Hello!",
+					}
+					respSendCR, err := aliceMessenger.SendContactRequest(context.Background(), request)
+					fmt.Println("[Alice] function SendContactRequest response: ", respSendCR)
```

Comment:
It should be possible to use `cCtx` instead of a Background() context. (Same in line 246, 287 and 320)